src/parser/parser.test.js

```diff
@@ -37,5 +37,11 @@ describe("parser", () => {
   
       expect(parse(tokens)).toMatchSnapshot();
     });
+
+    test("link", () => {
+      const tokens = tokenize("[foo](bar)");
```

Comment:
Si haces el parseo recursivo arriba, prueba con un texto recursivo en el link

```suggestion
      const tokens = tokenize("[***foo***](bar)");
```

Reply:
El caso ***foo*** de por si solo me bota el código debido a que es <p><em><strong>hola</strong></em></p> y <em> o Emphasis no están implementados aun.

Pero con  "[**foo**](bar)" funciona bien! se deja este ultimo. 😄